DigitalLearningSolutions.Web/Views/Shared/PaginatedPage/_PaginationNav.cshtml

```diff
@@ -14,7 +14,7 @@
       </div>
     </div>
     <div class="page-indicator-container">
-      <span aria-live="polite" aria-atomic="true">
+      <span role="alert">

```

Comment:
This was previously on `aria-live="polite"` which means it would only be read out when updated. Changing it to an alert role means it will always be announced on page load as well (both when redirected here via the new returnPage functionality, and also when first loading the page).

Reply:
If we have a filter cookie set, will it now get read out twice on page load? (Once when the HTML loads, and once when the pagination / filter code runs)

Reply:
We no longer read them out twice, even if we have a filter cookie set. The initial HTML load is read out, but the SearchSortPaginate function in JS that runs does not update the result count, so the value is not read out twice like it was before.